### ISSUE TITLE
Introduce community sprints

### ISSUE THREAD

**Issue by dagwieers** [ansible/community#171]
As part of community-building and getting more existing contributors involved in the development process, it could be worthwhile to introduce regular community sprints. They could be organized by existing Working Groups and would be very focused.

The main goals are:
- Close more issues and PRs in a timely fashion
- Get more people involved in the process and grow the community
- Create better standards while doing
- Increase common sense of achievement and improve team collaboration

Interested parties join on e.g. bluejeans to collaborate on closing issues/PRs. Either individually, in small groups or as a team, depending on the skill-level of the participants. It is required to have contributors with commit access involved in order to be able to close issues or merge PRs. This to unblock issues as they arise and have a smooth process.

Questions during this process can easily be shared and common solutions or new standards can be developed in a very short time-frame. Important decisions can be deferred to the Working Group meetings. New contributors learn the team's priorities and thinking in the process.

The goal would not necessarily be limited in time, but rather very concrete goals *"we do this for 10 issues or 10 PRs"*.

In order to prepare these sessions, we could make selections of issues/PRs in advance, e.g. based on labels (feature_idea, new_module, handle newer first, ...) and the original reporter/contributor could be notified of the fact that this sprint is happening and their involvement would be appreciated (indirectly they get involved in the process too and are more likely to become long-term contributors). *"Tuesday at 16:00PM UCT we will be discussing this ticket, please fix any remaining issues and let us know if you'd like to participate"*

The sprint could be ended by sharing a beer (each individually, prepared) and celebrate the achievement. Giving opportunity to discuss things learned or create an atmosphere where new ideas can be shared (typically newcomers bring in fresh ideas and question existing practices). Collaborators are more likely to stay until the end to take part in this celebration :-)

**Comment by gregdek** [comment]
Why are you assigning it to us? Just do it. It's a great idea.

Seriously, though, it seems like the folks from the Windows SIG can just say "hey we're doing this" and if it works well we just copy what you did, and I don't want to be stop-energy because you're waiting for permission or something. It's your SIG now, and you don't need our permission. :)

That said, if there's an actual work item you want from me, I'm happy to do it.

**Comment by dagwieers** [comment]
I assumed you guys had more experience with doing something like this.
And maybe this was something you wanted to discuss for other parts of the community too.
So it could become a wider initiative altogether. No worries, we'll discuss this on the next WWG meeting.

**Comment by gregdek** [comment]
I think it's definitely something we'll want to encourage more broadly in the other SIGs. 

One thing we could consider doing: setting up a standing BlueJeans meeting number for each of the SIGs, for use however/whenever that SIG wants to use it.

**Comment by bcoca** [comment]
This is what some of the core/windows/networking meetings end up being, looking/reviewing a few PRs and closing them/advancing them.

Not opposed to having more, just saying we can just add to the agendas of existing meetings which lately end early.

**Comment by dagwieers** [comment]
@bcoca: We do handle PRs that were put on the agenda, but there's no concerted effort to go over each of them, let alone issues. Fact is that the number of open Windows issues/PRs has been growing rapidly so the existing meetings won't help in that regard.

Now to attract new potential collaborators, not doing it during the meetings has the advantage that there is only one thing we are focusing on, in a concerted collaboration, using tools that are better fit for this job.

**Comment by gundalow** [comment]
With my Networking & Testing hats on, this is something we may look at in the future, though we will be watching with interest the Windows Sprints, and I'd like to sit in your first sprint..

**Comment by dagwieers** [comment]
We plan to do one on Tuesday 2017-07-04 somewhere in the late afternoon. Since US is on holiday, and jborean93 is offline, we can do it during CEST-friendly hours. We already defined the scope in the Wiki: https://github.com/ansible/community/wiki/Windows:-Sprints#2017-07-04-sprint-1

**Comment by gregdek** [comment]
FYI: https://twitter.com/gregdek/status/880479503797149696

**Comment by gregdek** [comment]
So @dagwieers I'm gonna go ahead and assign this to you. Whatever you want the resolution of this issue to be -- a HOWTO for running community sprints, maybe -- lmk. You can say "hey I'd like X" and reassign it to me, or a PR, or whatever you think best.
